Integrations/integration-nmap.yml

```
@@ -4,9 +4,10 @@ commonfields:
 description: Run nmap scans with the given parameters
 configuration: []
+toversion: 4.5.0
```

Better to put 4.5.9 incase we ever release a sub version
```suggestion
toversion: 4.5.9
```